src/helper.js

```diff
@@ -21,6 +21,7 @@ var Helper = {
 	password: {
 		hash: passwordHash,
 		compare: passwordCompare,
```

Comment:
Instead of creating separate async method, just update all the code to use async code.